puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/tfmROR52.groovy

```diff
@@ -0,0 +1,67 @@
+def packages_to_build
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+
+                checkout([
```

Comment:
Not saying it should change but what is the advantage of using this over git url: <git_url>, branch <branch_name> ?

Reply:
This form actually restricts this to run only on the branch specified via the internals.

Reply:
Ahh okay, good with me.